lms/djangoapps/discussion/tasks.py

```diff
@@ -113,8 +113,10 @@ def _is_not_subcomment(comment_id):
 
 def _is_first_comment(comment_id, thread_id):
     thread = cc.Thread.find(id=thread_id).retrieve(with_responses=True)
-    first_comment = thread.children[0]
-    return first_comment.get('id') == comment_id
+    if getattr(thread, 'children', None):
```

Comment:
Can we also have an `else` statement here with logging to print out the thread attributes when children is `none`? I want to know what threads would cause this condition.

Reply:
sure @schenedx